drivers/storage/isilon/executor/isilon_executor.go

```diff
@@ -43,6 +44,18 @@ func (d *driver) Name() string {
 	return isilon.Name
 }
 
+// Supported returns a flag indicating whether or not the platform
+// implementing the executor is valid for the host on which the executor
+// resides.
+func Supported(ctx types.Context, opts types.Store) (bool, error) {
+	// make sure mount.nfs is present
+	if err := exec.Command(
```

Comment:
This assumes that both the `which` and `mount.nfs` commands are in the `PATH` variable as provided to the executor. Is there perhaps a cleaner way to do this?